.github/workflows/nuget_publish.yml

```diff
@@ -13,21 +13,21 @@ jobs:
     runs-on: ubuntu-latest
     name: publish nuget
     needs: test
+    env:
+      VERSION_FILE_PATH: CSharpier.Build.props
+      TAG_FORMAT: "*"
+      NUGET_KEY: ${{secrets.NUGET_API_KEY}}
     steps:
       - uses: actions/checkout@v2
       - name: publish csharpier on version change
```

Comment:
maybe clarify to publish `cshapier core` on version change, as that's what it's now called.
Or maybe put one in brackets, to make easier for someone debugging who hasn't got the mental mapping of `CSharpier.csproj` to `CSharpier.Core`